From acb3269dd89c87d13c8e238ebb0c136e3824ac86 Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Thu, 15 Dec 2016 15:12:05 +0100
Subject: [PATCH] Add missing newline characters in g_debug

Unfortunately, g_debug uses custom handler in GVfs which requires
additional new line character. Commit a7b0a65 and commit 1d67090
changed g_warning to g_debug, but forgot to add additional new line
characters.
---
 daemon/gvfsbackendnetwork.c | 2 +-
 daemon/mount.c              | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/gvfsbackendnetwork.c b/daemon/gvfsbackendnetwork.c
index 95fee1d6..c6a103d1 100644
--- a/daemon/gvfsbackendnetwork.c
+++ b/daemon/gvfsbackendnetwork.c
@@ -358,7 +358,7 @@ recompute_files (GVfsBackendNetwork *backend)
           else
             {
 	      char *uri = g_file_get_uri (server_file);
-              g_debug ("Couldn't create directory monitor on %s. Error: %s",
+              g_debug ("Couldn't create directory monitor on %s. Error: %s\n",
 	     		 uri, error->message);
 	      g_free (uri);
               g_clear_error (&error);
-- 
2.15.0

